app/views/pages/get-started/contact-us-tutorial/index.liquid

```diff
@@ -0,0 +1,168 @@
+
+
+### Step 5: Authenticate your environment
+
+For pos-cli to know which instance is related to any particular codebase, you need to make a connection between those. Authenticate your environment:
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+pos-cli env add --url https://YOUR-INSTANCE.staging.oregon.platform-os.com staging
+</code></pre>
+
+A hidden `.pos` file has been created, and this is what pos-cli uses behind the scenes to authenticate.
+
+For further details, refer to the [Authenticate your environment](get-started/working-with-the-code-and-files/#authenticate-your-environment) guide.
+
+### Step 6: Authorize your instance
+
+Authorize your instance in the browser when prompted:
+
+{% # TODO: check size %}
+<img loading="lazy" class="w-full" src="{{ 'images/get-started/contact-us-tutorial/authorize.png' | asset_url }}" alt="Authorize button on the Partnerl Portal">
+
+You should receive the following message:
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+Environment https://contactus.staging.oregon.platform-os.com/ as staging has been added successfully.
+</code></pre>
+
+### Step 7: Install the platformOS core module
+
+Ensure that you are in the project root, then:
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+pos-cli modules install core
+pos-cli modules download core
+</code></pre>
+
+Read and follow the instructions in the installation section of the [platformOS core module documentation](https://github.com/Platform-OS/pos-module-core) for further details.
+
+#### Step 8: Upload your code to your instance
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+pos-cli deploy staging
+</code></pre>
+
+For further details, refer to the [Upload your code to an instance](/get-started/working-with-the-code-and-files/#upload-your-code-to-an-instance) guide.
+
+### Step 9: Enable code syncing
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+pos-cli sync staging
+</code></pre>
+
+For further details, refer to the [Upload code changes automatically when you save the file](/get-started/working-with-the-code-and-files/#upload-code-changes-automatically-when-you-save-the-file) guide.
+
+### Step 10: Start the pos-cli GUI
+
+Use platformOS’s database management tool, which is part of pos-cli. Start the pos-cli GUI and sync your environment with a single command:
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+pos-cli gui serve staging --sync
```

Comment:
If we are going to use the `--sync` param, then the previous step is not required as it will duplicate the syncing processes. Either `pos-cli sync staging` and `pos-cli gui serve staging` in a separate terminal window or just `pos-cli gui serve staging --sync`.

Reply:
Fixed the wording.